### DISCUSSION TITLE
Add copy button to markdown blocks

### DISCUSSION THREAD

**Discussion by grabani** [cline/cline#378] [Feature Requests]
Add "Copy" generated markdown (especially code). Expect to see copy button in the top right (see screenshot below):

![image](https://github.com/user-attachments/assets/4eb4c66b-8d4f-4326-8ab6-2dbe1b675e57)

**Comment by SinghChinmay** [comment]
I also want that, and also
Copy as .md the output which is there in the task.

When Cline completes a task, I want to copy some part or something, but the formatting is not there when doing the copy, it copies just text, I want to copy the formatted output shown in the Task Completed section and other sections.
